### DISCUSSION TITLE
My i3 version is stuck at 4.20.1, and it says that there are no more updates.

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#5848] [Q&A] [answered]
I want to customize my wm to have gaps, but it doesn't want to update and I don't really want to install i3-gaps.
![fetch](https://github.com/i3/i3/assets/154634737/7e322e02-c3c6-4fd0-8a88-8d4ac492f771)
![screenshot1](https://github.com/i3/i3/assets/154634737/1bddbf4d-5265-457b-a8ff-560d1540a521)

**Comment by orestisfl** [comment, marked as answer]
See https://i3wm.org/docs/repositories.html and https://github.com/i3/i3/discussions/5835

**Reply by ghost** [reply]
I never found that question, thanks!
